self_test_07.20.2019/FridgeStatusEventArgs.cs

```diff
@@ -0,0 +1,32 @@
+﻿using self_test_07._20._2019.Enum;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace self_test_07._20._2019
+{
+    public class FridgeStatusEventArgs : EventArgs
+    {
+        public SwitchStatus FridgeStatus { get; set; }
+
+        public DoorStatus FridgeDoor { get; set; }
+
+        public DoorStatus FreezerDoor { get; set; }
+
+        public string Message { get; }
+
+        public FridgeStatusEventArgs(string message)
+        {
+            this.Message = message;
+        }
+
+        public FridgeStatusEventArgs(SwitchStatus fridgeStatus, DoorStatus fridgeDoor, DoorStatus freezerDoor)
```

Comment:
This constructor is not used.

Reply:
I know, but at first I was going to use it. Constructor was deleted. Fixed.